head.html

```diff
@@ -1,5 +1,6 @@
+<meta http-equiv="Content-Security-Policy" content="default-src 'self'; script-src 'self' 'unsafe-inline' 'unsafe-eval' rum.hlx.page *.mktoweb.com *.googletagmanager.com js.zi-scripts.com *.zoominfo.com *.merative.com *.salesloft.com *.googleadservices.com *.kryogenix.org *.adsrvr.org www.youtube.com cm.everesttech.net documentservices.adobe.com googleads.g.doubleclick.net assets.adobedtm.com merative.tt.omtrdc.net cdn.jsdelivr.net dpm.demdex.net munchkin.marketo.net rtp-static.marketo.com sjrtp7.marketo.com cdn.cookielaw.org resources.digital-cloud-west.medallia.com *.kampyle.com js.driftt.com; img-src 'self' data: *.merative.com *.mktoweb.com *.google.co.in *.google.com *.google.ca *.googletagmanager.com *.112.2o7.net *.ytimg.com assets.adobedtm.com dpm.demdex.net cdn.cookielaw.org cm.everesttech.net *.medallia.com *.kampyle.com; frame-src 'self' *.mktoweb.com *.navattic.com *.adsrvr.org www.youtube.com player.vimeo.com *.merative.com documentservices.adobe.com merative.demdex.net *.medallia.com *.kampyle.com; connect-src 'self' *.mktoweb.com *.mktoresp.com *.marketo.com *.salesloft.com js.zi-scripts.com *.zoominfo.com *.adsrvr.org *.112.2o7.net assets.adobedtm.com viewlicense.adobe.io *.onetrust.com merative.tt.omtrdc.net analytics.google.com stats.g.doubleclick.net dpm.demdex.net cdn.cookielaw.org *.medallia.com *.kampyle.com; style-src 'self' 'unsafe-inline' *.mktoweb.com *.marketo.com go.merative.com assets.adobedtm.com cdn.cookielaw.org *.medallia.com *.kampyle.com; font-src 'self' *.mktoweb.com assets.adobedtm.com cdn.cookielaw.org *.medallia.com *.kampyle.com; object-src 'none'; manifest-src 'self'; media-src 'self' *.merative.com; worker-src 'none';">
```

Comment:
Do you mind changing the order of the 1st two entries (or something) just so we can confirm which CSP is taking precedence?

Reply:
You can have multiple CSPs and the URL has to pass through all the CSPs for it to work.
https://w3c.github.io/webappsec-csp/#multiple-policies